environment.yaml

```diff
@@ -0,0 +1,25 @@
+name: dropsonde_test 
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python>=3.10
+  - sphinx
+  - ca-certificates
+  - openssl
+  - myst-parser
+  - certifi
+  - sphinx-autobuild
+  - numpy
+  - nbsphinx
+  - pip>=20.1
+  - xarray
+  - netcdf4
+  - ipython
+  - pre-commit
+  - metpy
+  - pip:
```

Comment:
AFAIK, it is still considered good practice to explicitly list itself `pip` as a dependency when using pip dependencies